### DISCUSSION TITLE
manually set the size of the navigator view?

### DISCUSSION THREAD

**Discussion by nancyxyzou** [openseadragon/openseadragon#2498] [Q&A] [answered]
One of two viewers is initialized to span the entire width of the screen. after you press a button, this first viewer is shrunk to half the width of the screen to fit 2 identically sized viewers side-by-side. 
This might be why the navigators are lopsided, as the first one is the one with the larger navigator-- larger, but not double in size or the same size as the navigator appeared when first initialized to fit the screen either.

It just looks a little goofy, so I'm wondering if there's a way to manually set the size of the navigator.

**Comment by iangilman** [comment, marked as answer]
There are lots of options for sizing and positioning the navigator.  You can find them here: https://openseadragon.github.io/docs/OpenSeadragon.html#.Options. I know that's a long list... They start with `showNavigator`.

**Reply by nancyxyzou** [reply]
Thank you so much!
